pyannote/audio/cli/evaluate_config/config.yaml

```diff
@@ -1,5 +1,6 @@
 model: ???
 protocol: ???
+registry: ???
```

Comment:
Can we still make it optional?
Indeed, `pyannote.database` will automatically load `database.yml` in current directory.